docs/competition/teams/role-descriptions/reception.md

```diff
@@ -56,11 +47,9 @@ Volunteers will be turning up throughout the competition, it's important to make
 
 You should have the following items present for your duties. If you do not, please let the competition event coordinator know as soon as possible.
 
-* Spare Media Consent Forms (MCFs) to allow visitors to enter.
-* Pens to fill in forms.
-* Wristbands to give to entrants.
+* Pens.
 * Competitor booklets to give to teams.
-* This documentation and flow chart.
+* This documentation.
```

Comment:
Also a copy of the safeguarding policy, plus whatever is needed to record that volunteers have signed it?

Related to that: how will reception know who has already signed it ahead of time?